### PR TITLE
Make page title use controller title if available

### PR DESCRIPTION
**Pull request by anildash**

### CODE REVIEW

File: webapp/_lib/view/_header.tpl [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en" prefix="og: http://ogp.me/ns#" itemscope itemtype="http://schema.org/Article">
 <head>
     <meta charset="utf-8">
-    <title>{if $controller_title}{$controller_title} | {/if}{$app_title}</title>
```
Comment (ginatrapani) [inline webapp/_lib/view/_header.tpl]: In this deleted line, the page uses $controller_title if it is set. Why the change?

**Comment by anildash** [comment]
No idea. I read this as exactly backwards. Less coding in the wee hours!
